pkgs/tools/system/logcheck/default.nix

```diff
@@ -1,13 +1,12 @@
 { stdenv, fetchurl, lockfileProgs, perl, mimeConstruct }:
 
 stdenv.mkDerivation rec {
-  _name   = "logcheck";
-  version = "1.3.17";
-  name    = "${_name}-${version}";
+  version = "1.3.18";
+  name    = "logcheck_${version}";
```

Comment:
The _ is what is correct. http://http.debian.net/debian/pool/main/l/logcheck/logcheck_1.3.18.tar.xz is the source from Debian